[from now] 2016/11/13 20:37:27

slushpupie opened the pull request:
diff --git a/introduction.md b/introduction.md
index 023ea01..07c744d 100644
--- a/introduction.md
+++ b/introduction.md
@@ -3,3 +3,8 @@
 This is an introduction

 Some mention about the Ruby path
+
+ * Java
+ * Git
+ * Linux
+ * Grue Slaying

slushpupie committed:
[from now] 2016/11/13 20:37:27

diff --git a/introduction.md b/introduction.md
index 023ea01..07c744d 100644
--- a/introduction.md
+++ b/introduction.md
@@ -3,3 +3,8 @@
 This is an introduction

 Some mention about the Ruby path
+
+ * Java
+ * Git
+ * Linux
+ * Grue Slaying

codeschool-kiddo commented on introduction.md:
```diff
@@ -3,3 +3,8 @@
 This is an introduction
```
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

slushpupie committed:
[from now] 2016/11/13 20:39:30

diff --git a/introduction.md b/introduction.md
index 07c744d..9b66bc6 100644
--- a/introduction.md
+++ b/introduction.md
@@ -8,3 +8,4 @@ Some mention about the Ruby path
  * Git
  * Linux
  * Grue Slaying
+ * Killing history using git rebase
